### ISSUE TITLE
Semi-false positive in class 

### ISSUE THREAD

**Issue by Page-** [za-creature/coffeescope#19]
``` coffee
->
    return class X
```

gives

```
#2: Variable "X" is never used.
```

which is technically accurate, a variable X is created and never used, but my reason for naming the class is in order to have any stack traces include a nice name rather than `_Class`.

For reference the generated code is:

``` js
function() {
  var X;
  return X = (function() {
    function X() {}

    return X;

  })();
}
```

vs

``` js
function() {
  return (function() {
    function _Class() {}

    return _Class;

  })();
}
```

without the `X`

**Comment by za-creature** [comment]
I'm going to table this as it's not really a bug and the other issues are prioritary. For now, your options are:

``` coffeescript
->
  return class X  # noqa
    foo: ->
      bar
```

or

``` coffeescript
->
  class X
    foo: ->
      bar
  return X
```

**Comment by za-creature** [comment]
Landed in 0.3.5 via the `unused_classes` option (defaults to true but you can change it to false to disable this check)

**Comment by Page-** [comment]
I agree it isn't/wasn't a high priority item at all, it was mostly a case I came across whilst adding linting throughout our projects.  Thanks for fixing it so quickly regardless :D
